zcash_client_backend/proto/proposal.proto

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2023 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+syntax = "proto3";
+package cash.z.wallet.sdk.ffi;
+
+// A data structure that describes the inputs to be consumed and outputs to
+// be produced in a proposed transaction.
+message Proposal {
+    uint32 protoVersion = 1;
+    // ZIP 321 serialized transaction request
+    string transactionRequest = 2;
```

Comment:
What is the relation of this to the other fields? It seems redundant, and could be inconsistent with other fields.

Reply:
This is not at all redundant; the transaction request is what encapsulates all of the information about transaction recipients.